target/linux/ath79/image/generic.mk

```diff
@@ -341,3 +341,13 @@ define Device/wd_mynet-wifi-rangeextender
   SUPPORTED_DEVICES += mynet-rext
 endef
 TARGET_DEVICES += wd_mynet-wifi-rangeextender
+
+define Device/rosinson_wr818
+  ATH_SOC := qca9563
+  DEVICE_TITLE := ROSINSON WR818
+  IMAGE_SIZE := 15936k
+  IMAGES := sysupgrade.bin
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
```

Comment:
This is almost the same as the default sysupgrade.bin build command (in image/Makefile). I think it's not necessary to save a few kilobytes by removing the "pad-to BLOCKSIZE" in the build command.

Reply:
@981213 Thanks a lot for the review. I like to get rid of the `pad-to BLOCKSIZE` for all ath79 boards, as it isn't really required.

But since I don't have all boards to test and ensure that it works without the extra padding, I encourage contributor to add the line without the padding.

On the long run run we should have more boards without the extra padding, can change the defaults to no padding and add the padding only for the early added boards. You get the point.